lib/carrierwave/neo4j.rb

```diff
@@ -210,4 +208,4 @@ def store_previous_changes_for_#{column}
   end
 end
 
-Neo4j::ActiveNode.send :include, CarrierWave::Neo4j
+Neo4j::ActiveNode.include CarrierWave::Neo4j
```

Comment:
You created a sample app and tested this out to make sure it works?

Reply:
I just ran tests. Tests don't catch this ?

Reply:
No. Remember? I called this out very explicitly:

https://github.com/neo4jrb/carrierwave-neo4j/pull/17#discussion_r419090541

If you want to make meta-programming changes to this (_unbelievably_ fragile) gem, please click-test them by standing up a sample app and testing by hand. This fragility is inherent in the way `carrierwave` was written so there isn't really a way to fix it at the `neo4j.rb` level.

As I also mentioned before: I tried to clean up some of the original meta-programming done by @cheerfulstoic (including this change to `.include`) but I couldn't get in touch with him to confirm why things were structured the way they were. The `carrierwave` upstream codebase is sort of an early-2000s Ruby runtime meta-programming trash fire... I assume that's why Brian needed to do some of the things he did -- and also why the test harness can't faithfully recreate objects in MRI which behave in an identical way to how they behave inside a real Rails app. Because `Neo4j::ActiveNode` is a module instead of an abstract class, it wasn't possible for me to follow the ActiveRecord example here, either.

Sorry about this. I know it's a hassle... the week I spent bringing the gem back to life is a week of my life I'll never get back. I feel your pain. :face_with_head_bandage:

Reply:
I just moved our app to your branch and tested this out (as thoroughly as I could think to do, anyway)... it certainly didn't break in any of the obvious ways that my changes to that `send :include` line broke things when I was working on the gem, so hopefully everything is kosher.

The change makes me really nervous and I'm trying to instill that nervousness in you, @amitsuryavanshi  :wink: ...if it turns out that this gem is having weird behaviour in the future, let's please remember to revisit this.

Just for my own sanity, I assume the change linked below is what makes the call to `.include` possible? I'm pretty sure I didn't do this when I tried this change originally and if you know this is what makes it work, the whole thing would freak me out a lot less since we'd be riding on "Amit knows this is the right way to do it" rather than "Steven tried it in an app and it didn't seem to break." :stuck_out_tongue_winking_eye: 

https://github.com/pariyatti/carrierwave-neo4j/pull/1/files/1314f41a4d5501c5833e5a91b9a98afb38e9b3bf#diff-823bbce648661b3ecda19b60e9731b65L106

Reply:
Pardon me, lot of things on my plate, forgot your comment on other PR. I assume we should be safe with these changes.